phoenix-core/src/main/java/org/apache/phoenix/util/MetaDataUtil.java

```diff
@@ -303,6 +303,54 @@ public static long getSequenceNumber(List<Mutation> tableMetaData) {
         return getSequenceNumber(getPutOnlyTableHeaderRow(tableMetaData));
     }
     
+    /**
+     * Returns the sequence number of the parent table if we have a mutation that is updating the
+     * parent table's encoded column qualifier
+     */
+    public static long getParentSequenceNumberForAddColumnMutations(List<Mutation> tableMetaData) {
+        byte[] parentTableRowKey = null;
+        for (Mutation tableMutation : tableMetaData) {
```

Comment:
I don't think we really need this.

Reply:
Done. We don't need this because we acquire a mutex on the client side which will prevent concurrent changes to the encoded col qualifier counter.